Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
+		- Learn to control the plane with what you have; don't forget trim can help; make a precautionary landing
+	- A door opens - Deal with it in the air, or land normally
+	- Low oil pressure - Could lead to engine failure.  Precautionary landing
+	- Low fuel - Divert to alternate to refuel
+	- Low battery - Shed as much load as possible; estimate time available; land accordingly
+	- Retractable gear malfunction - (out of scope for this lesson)
+	- Flew into a cloud, or lost minimum VFR visibility
+		- Rely on [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] skills; reverse course; request help on the radio
+	- Icing - Reverse course or choose a different altitude; request help on the radio
+
+##### Key Procedures in Emergencies
+These are the emergency procedures referenced in the section above.
+
+1. Emergency Descent:
+	- Goal is to rapidly descend for emergency landing, and possibly put the fire out.
+	- **(POH)**
+		- Cut off fuel flow
+		- Start descent with 30°-45° bank, airspeed >= 100kts.  Bank away from the fire so the flames don't cover the windshield
+		- Once fire is out, execute Forced Landing.  See [[SOLO.15 Emergency Landings]]
+2. Forced Landing:
+	- **(POH)** See [[SOLO.15 Emergency Landings]]
+3. Precautionary landing: (By definition, assumes engine power is available)
+	- See **(POH)** procedure
+4. Powering up critical items after an electrical fire or burning smell:
+	- **(POH)** Turn off the master and stby switches.  Turn off all electrical switches.  Note any popped breakers, but do not reset. Turn on the master.  Turn on electrical switches for avionics or other required equipment, one at a time, waiting for signs of burning.
+
+##### Other important topics for emergencies
+1. Emergency equipment and survival gear
+	1. [[ACS]] IX.C mentions: ELT.  Fire extinguisher operation. Survival gear (food, water, shelter).  Overwater operations.
+2. Emergency squawk codes: [[Squawk 7700]], [[Squawk 7600]], [[Squawk 7500]]
 > [!caution] Even passing through one of these codes will trigger an alert for [[ATC]]. As you set a squawk code, be mindful to not pass through one accidentally.
-9. [[Intercept Procedures]]
-10. Common errors
-	1. Failure to use flow patterns or memory items
-	2. Failure to use checklists, or improper use
+3. [[Intercept Procedures]]
+4. Ballistic parachutes
+5. Auto-land systems
+
+##### The Checkride
+On the checkride, the examiner wants to see you demonstrate a Forced Landing and an Emergency Descent.  They'll trigger these scenarios by saying something like:
+- "Oops, you just lost the engine"   => Your cue to demonstrate a Forced Landing ([[SOLO.15 Emergency Landings]])
+- "I see flames on the right side of the cowling"
+	- This is your cue to demonstrate an Emergency Descent
+	- If the examiner then says "the flames are out", transition to a Forced Landing
+
+##### Common errors
+1. Failure to use flow patterns or memory items
+2. Failure to use checklists, or improper use
```

Comment:
3. Not having checklist handy while flying -- you shouldn't need to dig for this!